content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available all Team Pro and Enterprise customers to preview."
+meta_image: meta.png
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News","Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `pulumi-policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks.
+
+<!--more-->
+
+You can run policies against any Pulumi stack, independent of the stack's language. This means you can have policies written in TypeScript and run them against a Python stack. Policies are grouped in Policy Packs and run against a stack during both `pulumi preview` and `pulumi up`. Each resource in a stack is run through the Policy Pack prior to creation, meaning you can block the creation of out-of-compliance resources.
+
+## Why
+
+Using Pulumi, developers and operators are empowered to self provision their infrastructure and move quickly. One our core goals is to allow you to codify best practices and share them with your team. We often hear from organization leaders that they're concerned about how to make sure their team doesn't accidentally go off the rails. CrossGuard provides guardrails your leadership needs to allow your team to continue to operate self-sufficiently.
+
+## Key Features
+
+The key features available during the CrossGuard preview are:
+
+* TypeScript `pulumi-policy` SDK to express custom policies
```

Comment:
For each of these bullets, it seems like we should have a link. Either to the GitHub repo's or to the relevant documentation.

Reply:
Was able to add one to all except the last one since thats an org / update specific URL